README.md

```diff
@@ -1,2 +1,3 @@
 # massa-wallet-provider
+
```

Comment:
please fill the readme

Reply:
Will add once you and greg approve the standard.

Reply:
I have added some now.